iroh-bytes/src/provider.rs

```diff
@@ -672,7 +672,7 @@ pub async fn send_blob<W: AsyncWrite + Unpin + Send + 'static>(
             debug!("done sending blob {} {:?}", name, res);
             res?;
 
-            Ok((SentStatus::Sent, size))
+            Ok((SentStatus::Sent, 0))
```

Comment:
why is this `0`?

Reply:
I didn't have the size at some point. I have it again now, so no I will put it in.